iOS/DoraemonKit/Src/Core/Plugin/Performance/CPU/Function/DoraemonCPUUtil.m

```diff
@@ -8,11 +8,17 @@
 #import "DoraemonCPUUtil.h"
 #import <mach/mach.h>
 #import <UIKit/UIKit.h>
+#import "DoraemonBacktraceLogger.h"
 //#import <sys/sysctl.h>
 
 @implementation DoraemonCPUUtil
 
 + (CGFloat)cpuUsageForApp {
+    DoraemonCPUUsageCallStackModel *model = [self cpuUsageAndCallstackForThread];
```

Comment:
The cpu samples at a rate of 0.5 seconds， we don't need  fetch call stack with each sample,  get callstack  when cpu usage is anomaly，

Reply:
sure.